docs/api/remote-procedure-calls-raw-transactions.md

```diff
@@ -1144,6 +1144,68 @@ Result:
+| ---- | ------------ | ----------------------- | ----------- |
+| Transactions | object | Required<br>(exactly 1) | A JSON object with block hashes as keys and lists of transaction hashes as values (no more than 100 in total) |
+|→<br>Block Hash | array | Required<br>(1 or more) | The block hash and the list of transaction ids to fetch |
+|→ →<br>Transaction ID | string | Required<br>(1 or more) | A transaction ID |
+
+_Parameter #2---whether to get the serialized or decoded transaction_
+
+| Name    | Type | Presence             | Description |
+| ------- | ---- | -------------------- | ----------- |
+| Verbose | bool | Optional<br>(0 or 1) | Set to `false` (the default) to return the serialized transaction as hex.  Set to `true` to return a decoded transaction in JSON. |
+
+_Result (if transactions not found)---`null`_
+
+| Name     | Type | Presence                | Description |
+| -------- | ---- | ----------------------- | ----------- |
+| `result` | null | Required<br>(exactly 1) | If no transactions were found, the result will be JSON `null`.  This can occur because the transactions don't exist in the block chain or memory pool, or because it isn't part of the transaction index.  See the Dash Core `-help` entry for `-txindex` |
+
+_Result (if verbose=`false`)---the serialized transactions_
+
+| Name     | Type         | Presence                | Description |
+| -------- | ------------ | ----------------------- | ----------- |
+| `result` | object | Required<br>(exactly 1) | If the transaction was found, this will be an object containing the serialized transaction encoded as hex. |
+|→<br>TXID / Raw tx | string:string | Required<br>(1 or more) | A key/value pair with the transaction ID (key) and raw transaction data (value). See the [`getrawtransaction` RPC](../api/remote-procedure-calls-raw-transactions.md#getrawtransactionmulti) for an example of the hex transaction data. |
+
+_Result (if verbose=`true`)---the decoded transactions_
+
+| Name                        | Type           | Presence                | Description |
+| --------------------------- | -------------- | ----------------------- | ----------- |
+| `result`                    | object         | Required<br>(exactly 1) | If the transaction was found, this will be an object describing it |
+|→<br>TXID / Decoded tx | string:string | Required<br>(1 or more) | A key/value pair with the transaction ID (key) and decoded transaction data represented in JSON (value). See the [`getrawtransaction` RPC](../api/remote-procedure-calls-raw-transactions.md#getrawtransactionmulti) for an example of the decoded transaction data. |
+
+_Examples from Dash Core 20.1.0_
+
+Transactions in serialized transaction format:
+
+```bash
+dash-cli getrawtransactionmulti '{"0000014b04deb0fe0884d3ca81a6239016c2a4838cd4b34494858630e457c62c": ["0667ab03bebb827057fcd55285d68dc10f30ef07dd04d1468c9f25888c22333c", "5c278dd4ca54b9c14789357f0311bcbf1b6d3182bcc76d229cb15cfbe6255ed0"]}' false
```

Comment:
missing `-testnet` or doesn't matter for sack of example?

Reply:
Good catch. In this case it's good to have it because then someone could actually run the example on their own node.

Reply:
Fixed this